<?xml version="1.0"?>
<launch>
  <!-- Example launch file for the PhaseSpace ImpulseX2E motion-tracker
       with ROS. Connects to the PhaseSpace OWL server, publishes a 
       world tf frame, and starts rviz.
  -->
<!--  <param name="robot_description" command="$(find xacro)/xacro.py '$(find sr_hand)/model/robots/xacro/arm_
and_hand_motor.urdf.xacro'"/>
  -->


  <!--
  <node pkg="tams_phasespace" name="phase_space_markers" type="tams_phasespace" output="screen">
    <param name="parent_frame" type="string" value="world" />
    <param name="owl_frequency" type="double" value="480.0"/>
    <param name="owl_server_name" type="string" value="192.168.169.201"/>
    <param name="debug_level" type="int" value="3" />
    <param name="publish_cameras" type="boolean" value="true" />
    <param name="marker_radius" type="double" value="0.05" />
  </node>
  -->



  <node pkg="tams_phasespace" name="calibration_visualizer" type="calibration_visualizer" output="screen">
    <param name="debug_level" type="int" value="3" />
    <param name="parent_frame" type="string" value="world" />
    <param name="marker_radius" type="double" value="0.05" />
   
    <param name="update_rate" type="double" value="480.0" />
    <param name="visualization_rate" type="double" value="1.0" />
    <param name="colormap_scale" type="double" value="0.01" />

    <remap from="owl_markers" to="phase_space_markers/led_markers" />
  </node>


  <node pkg="tf" type="static_transform_publisher" name="world_broadcaster" 
        args="1 0 0 0 0 0 world phasespace_base 100" />

<!--  <arg unless="$(arg config)" name="command_args" value="" />
  <arg     if="$(arg config)" name="command_args" value="-d $(find taser_moveit)/launch/moveit.rviz" />
-->


  <node name="rviz" pkg="rviz" type="rviz" respawn="false" output="screen">
    <!-- <rosparam command="load" file="$(find tams_phasespace)/config/kinematics.yaml"/> -->
  </node>

</launch>
